v1.5/reference/etcd/structure.md

```diff
@@ -1,290 +0,0 @@
----
```

Comment:
Same with this - I think we should be leaving the 1.5 docs in place.

We should move any tiered policy stuff out since that isn't actually supported, but we should be maintaining this and other 1.5 docs.